src/classes/UTIL_RecordTypes_TEST.cls

```diff
@@ -40,47 +40,40 @@ private class UTIL_RecordTypes_TEST {
     // TEST
     static testmethod void testRecTypes() {
         // try bogus values
-        Id reallyBogus = UTIL_RecordTypes.GetRecordTypeId('Bogus', 'Bogus');
```

Comment:
I dropped this line because with the change to SObjectType in the signature, there isn't a way to pass in a "bogus" object type (which is one of the main benefits to using SObjectType instead of string literals)